backendCore/dl/dl_model_parser.py

```diff
@@ -0,0 +1,84 @@
+from sklearn.tree import *
+from sklearn.ensemble import *
+from sklearn.linear_model import *
+from sklearn.svm import *
+from sklearn.naive_bayes import *
+import torch
+import torch.nn as nn
+from torch.autograd import Variable
+import torchvision.transforms as transforms
+
+
+def parse_deep_user_architecture(user_model):
+    """
+    new:
+    Each layer in the user_model has this format:
+    {"value": "LAYER_NAME", "parameters": [PARAMETER_1, PARAMETER_2, ...]}
+
+    Eg: [{"value": "LINEAR", "parameters": [50, 10]}] should become [nn.Linear(in_features=50, out_features=10)]
+
+    Args:
+        user_model (list): string form where you have the model name and the values for each parameter
+    Returns:
+        parsed_modules (list): list of nn.Module objects parsed
+    ----------------------------------------------------------------------------------------------
+    old:
```

Comment:
we should probably remove any old versions of the pydocs for methods we modify. I think we can keep the "new:" shown in this PR since that's the new data representation. 

Good catch!